week-1/schedule.md

```diff
@@ -0,0 +1,68 @@
+# Teamwork & Toolkit Week Schedule
+
+## Day 1
+
+| Time    | Activity                                                                                                                                                                                                                                                                                                                                | Learning outcomes |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| 09:45   |                                                                                                                                                                                                                                                                                                                                         |                   |
```

Comment:
Could we add 'Intro reading' here?

Reply:
I think on day 1 of week 1 it'd be nice if people can spend this time chatting and getting to know each other.

Reply:
Ah yeah that's a better idea